woocommerce/woocommerce
# Move LYS coming soon initialization to core profiler flow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Moved setting up coming soon options to core profiler flow.



### How to test the changes in this Pull Request:


#### Test with an existing site

1. Set up a fresh JN site
2. Go to Plugins > Add New Plugin, install and activate [WooCommerce 8.7.0](https://href.li/?https://github.com/woocommerce/woocommerce/releases/download/8.7.0/woocommerce.zip).
3. On activation, it should redirect you to profiler. Complete all steps.
4. Get the build from this branch and install it in JN.
5. Go to WooCommerce > Home
6. Observe the site visibility badge is set to `Live`

#### Test with a fresh site

1. Set up a fresh site in a JN with WCAdmin Test Helper
2. Go to onboarding wizard
3. Either complete or skip the onboarding wizard
3. In homescreen, observe your site is `coming soon`

#### Test tracks

1. After testing with a fresh site, go to WooCommerce > Status > Logs and look for `tracks.log`
2. Open it and look for `wcadmin_launch_your_store_initialize_coming_soon`
3. Observe it has the props for `coming_soon`, `store_pages_only`, and `private_link`

#### Test core upgrade

1. Go to Tools > WCA Test Helper > Tools
2. Run `Trigger WCA Install` and observe no error
3. Run `Run version update callbacks` with the latest database version selected and observe no error



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Move LYS coming soon initialization to core profiler flow

#### Comment 

</details>
